dagshub/data_engine/model/query_result.py

```diff
@@ -401,6 +404,78 @@ def download_binary_columns(
             num_proc=num_proc,
         )
 
+    def predict_with_mlflow_model(
+        self,
+        repo: str,
+        name: str,
+        version: str = "latest",
+        pre_hook: Callable = lambda x: x,
+        post_hook: Callable = lambda x: x,
```

Comment:
Just so it's clear that you have one input and one output
```suggestion
        pre_hook: Callable[[Any], Any] = lambda x: x,
        post_hook: Callable[[Any], Any] = lambda x: x,
```